moduleframework/common.py

```diff
@@ -91,6 +91,7 @@
 MODULE_DEFAULT_PROFILE = "default"
 TRUE_VALUES_DICT = ['yes', 'YES', 'yes', 'True', 'true', 'ok', 'OK']
 OPENSHIFT_INIT_WAIT = 50
+STATIC_CHECKS = ['static_dockerfiule_checks', 'helpmd_lint']
```

Comment:
wouldn't be better to add there some same pattern?
like you have: ``static_docker...`` and so that remane also helpmd_lint to same pattern.
otherwise there is typo ``...fiule...``